src/datachain/data_storage/warehouse.py

```diff
@@ -869,23 +878,29 @@ def update_node(self, node_id: int, values: dict[str, Any]) -> None:
 
     def create_udf_table(
         self,
-        name: str,
         columns: Sequence["sa.Column"] = (),
+        name: Optional[str] = None,
```

Comment:
Name is `udf_table_name()` in all `create_udf_table()` calls so let it be optional.